specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -459,12 +592,21 @@ namespace Microsoft.Windows.PushNotifications
         // Unregister any activator if present using a token and registrationKind
         // 1) If kind = PushTrigger is specified, the trigger itself will be removed
         // 2) If kind = ComActivator is specified, the Project Reunion Background Task component will no longer act as an InProc COM Server
-        static void UnregisterActivator(PushNotificationRegistrationToken token, PushNotificationRegistrationKind kind);
+        // 3) If kind = ProtocolActivator is specified, the application will be unregistered from the PushNotificationsLongRunningTask
+        static void UnregisterActivator(PushNotificationRegistrationToken token, PushNotificationRegistrationOptions options);
```

Comment:
If we don't need a token for unpackaged, probably we need an overload of this function.

Just my opinion, not sure what the opinion of Jon or Howard would be on this case.